presto-main/src/main/java/com/facebook/presto/metadata/StaticFunctionNamespace.java

```diff
@@ -51,6 +51,7 @@
 import com.facebook.presto.operator.aggregation.MaxDataSizeForStats;
```

Comment:
It seems like the last two commits should be one commit.  It's not clear to me why they're separate

Reply:
Jiexi asked if it would be possible to have a commit prior to the refactoring, so that's why the Original Tdigest commit is there. Will change the name to something more descriptive soon